# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gv/anti-alias-default.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- gv-3.5.8/source/gv_user_res.dat.orig	2003-12-05 19:31:42.000000000 +0100
+++ gv-3.5.8/source/gv_user_res.dat	2003-12-05 19:34:06.000000000 +0100
@@ -8,7 +8,7 @@
 GV.fallbackOrientation:	portrait
 GV.swapLandscape:	False
 GV.autoCenter:		True
-GV.antialias:		False
+GV.antialias:		True
 GV.respectDSC:		True
 GV.ignoreEOF:		True
 GV.confirmPrint:	True
